pipeline-steps/common.groovy

```diff
@@ -208,4 +208,19 @@ def gen_instance_name(){
   return instance_name
 }
 
+def writePyraxCfg(Map args){
+  cfg = """[rackspace_cloud]
+username = ${args.username}
+api_key = ${args.api_key}
+"""
+
+  tmp_dir = pwd(tmp:true)
```

Comment:
@melissa-kam you pointed out before the issue with writing secret files to the workspace (since the workspace can be visible through the UI) -- any concerns writing this to the temporary space?